### PR TITLE
Schema change

### PR DESCRIPTION
**Pull request by Wibblist**
-Added markers property to animation object
-Added marker object to helpers

### CODE REVIEW

File: docs/schema/shapes/polystar.json [hunk 1]
```diff
@@ -42,6 +41,11 @@
                     "description": "Star type, `1` for Star, `2` for Polygon",
                     "$ref": "#/$defs/constants/star-type",
                     "default": 1
+                },
+                "d": {
```
Comment (mbasaglia) [inline docs/schema/shapes/polystar.json:45]: polystar should already have `d` by getting it from [shapes/shape](print(speed))

File: docs/schema/animation/animation.json [hunk 2]
```diff
@@ -76,9 +80,24 @@
...
+                },
+                "mb": {
+                    "title": "Motion Blur",
+                    "$ref": "#/$defs/animation/motion-blur"
                 }
             },
             "required": ["w", "h", "fr", "layers", "op", "ip"]
+        },
+        {
+            "$ref": "#/$defs/animation/composition"
```
Comment (mbasaglia) [inline docs/schema/animation/animation.json:100]: I'm a bit confused by the diff here, it shows some changes that are already there. The merge request shows conflicts too so maybe it's based on an older commit?

Could you merge with the latest main?

File: docs/schema/shapes/polystar.json [hunk 3]
```diff
@@ -24,7 +24,6 @@
                     "$ref": "#/$defs/animated-properties/value"
                 },
                 "os": {
-                    "title": "Outer Roundness",
                     "title": "Outer Roundness as a percentage",
```
Comment (mbasaglia) [inline docs/schema/shapes/polystar.json:27]: I think here would be better to leave the first title and change the second into a `description`

**Review by mbasaglia — CHANGES_REQUESTED** [review]

File: docs/schema/animation/animation.json (hunk 2 above)
Comment (Wibblist) [reply docs/schema/animation/animation.json:100]: Yeah, hold on
I was just trying to add makers.

**Pull request closed without merging**